js-api-doc/compile.d.ts

```diff
@@ -37,18 +37,113 @@ export interface CompileResult {
   sourceMap?: RawSourceMap;
 }
 
+/**
+ * The result of creating a synchronous compiler. Returned by
+ * {@link initCompiler}.
+ *
+ * @category Compile
+ */
+export class Compiler {
+  /**Throws an error if constructed directly, instead of via
+   * {@link initCompiler}. */
```

Comment:
Properly format this comment

Reply:
Fixed in https://github.com/sass/sass/pull/3744/commits/11f1af3748befbd7e21e7a6e5cce55caf2b4ff8a